docker-compose.dev.yml

```diff
@@ -31,4 +34,58 @@ services:
       - '27017:27017'
     volumes:
       - './data/mongodb:/data/db'
-      - './logs:/var/log'
+      - './logs/mongodb:/var/log'
+
+  temporal:
+    image: temporalio/auto-setup:1.27.0
+    depends_on:
+      - temporal-db
+    environment:
+      DB: 'postgres12'
+      DB_PORT: '5432'
+      POSTGRES_USER: 'temporal'
```

Comment:
Does this mean that in production we would have MongoDB as well as PostgresDB? Seems not desireable.